modules/lightstep/src/main/scala/LightstepSpan.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package natchez
+package lightstep
+
+import cats.effect.{ Resource, Sync }
+import cats.implicits._
+import io.{ opentracing => ot }
+import io.opentracing.propagation.{ Format, TextMapAdapter }
+
+import scala.collection.JavaConverters._
+
+private[lightstep] final case class LightstepSpan[F[_]: Sync](
```

Comment:
What here is lightstep specific? This whole file is `opentracing` related, but vendor neutral, isn't it?

Reply:
Yes, that's why I mentioned it in the PR description. Having in mind the current library API, I'd probably extract them outside of vendor-specific implementations.

Reply:
Ah, I somehow missed the `JaegerSpan` comment, thanks for pointing it out. I would definitely vote for having it extracted to a single place, to also make it clear that it could be used for more OpenTracing vendors in the future.

Reply:
The problem here is that Jaeger and LightStep use different, incompatible versions of OpenTracing; so we would need at least two OpenTracing layers here. Maybe they'll get their act together at some point but for now I think it's ok to bury OpenTracing and just do vendor modules.

This is all out the window when OpenTelemetry shows up.